pandas/tools/tests/test_util.py

```diff
@@ -93,6 +93,11 @@ def test_set_locale(self):
             raise nose.SkipTest("Only a single locale found, no point in "
                                 "trying to test setting another locale")
 
+        if all(x is None for x in CURRENT_LOCALE):
+            # Not sure why, but on some travis runs with pytest,
```

Comment:
Yeah, very strange. Wasn't able to find anything about why that's happening, but that test / the `.setlocale` contextmanager relies on `getlocale()` being valid initially. I've started a list of followup issues, will add this.